SAK-48101 - Plus: TA comes across with access perms and is listed as a student in the GB (#11064)
sakaiproject/sakai: basiclti/docs/LTIROLES.md

over time.   The overrides allow for a quick response when it might take a little while before
we can fix Sakai and you get an upgraded version.

Per-Tool Outbound Role Mapping
------------------------------

There are special situations where a particular tool needs some very particular role
mapping or your Sakai sites have additional roles beyond the out-of-the-box roles.
When you are installing an LTI tool into Sakai, you have an option for mapping a particular

> Sakai role to a particular LTI role using a set of mapping strings that specify a Sakai
> role and the corresponding LTI role:

Reviewer: ```suggestion
Sakai role to a particular LTI role using a set of mapping strings, like this:
```
